content/docs/guides/cli.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "OSM CLI"
-description: "This section describes installing and using the `osm` CLI."
+title: 'OSM CLI'
+description: 'This section describes installing and using the `osm` CLI.'
```

Comment:
Same comment as before. Revert this if it's the same as using double quotes, for consistency.